src/fmu/dataio/dataio.py

```diff
@@ -659,21 +657,21 @@ def _validate_and_establish_fmucontext(self) -> None:
         elif not self._fmurun:
             logger.warning(
                 "Requested fmu_context is <%s> but since this is detected as a non "
-                "FMU run, the actual context is force set to non-fmu",
+                "FMU run, the actual context is force set to None",
                 self.fmu_context,
             )
-            self.fmu_context = FmuContext.NON_FMU
+            self.fmu_context = None
```

Comment:
I wish this didn't have to be None-able... but overall using the enums as part of the data model is much better